RockPaperScissors/RockPaperScissors/RockPaperScissorsGame.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RockPaperScissorsGame.swift
+//  RockPaperScissors
+//
+//  Created by maxhyunm, Mary on 2023/05/02.
+//
+
+struct RockPaperScissorsGame {
+    enum Menu: String {
+        case scissors = "1"
+        case rock = "2"
+        case paper = "3"
+        case exit = "0"
+        
+        var winningOpponent: Menu {
+            switch self {
+            case .scissors:
+                return .paper
+            case .rock:
+                return .scissors
+            case .paper:
+                return .rock
+            case .exit:
+                return .exit
+            }
+        }
+    }
+
+    enum Result: String {
+        case win = "이겼습니다!"
+        case lose = "졌습니다!"
+        case draw = "비겼습니다!"
+    }
+
+    func startGame() {
+        do {
+            let menu = try inputMenu()
+            
+            guard menu != .exit else {
+                print("게임 종료")
+                return
+            }
+            let gameResult = try playGame(menu)
+            
+            print(gameResult.rawValue)
+            
+            if gameResult == .draw { startGame() }
+        } catch GameError.menuNotFound {
+            print("\(GameError.menuNotFound.errorMessage)")
+            startGame()
+        } catch {
+            print("\(GameError.unknownError.errorMessage)")
+        }
+    }
+
+    private func inputMenu() throws -> Menu {
+        print("가위(1), 바위(2), 보(3)! <종료 : 0> :", terminator: " ")
+        
+        guard let inputValue = readLine(),
+              let menu = Menu(rawValue: inputValue)
+        else { throw GameError.menuNotFound }
```

Comment:
일반적으로 중괄호 뒤에서 개행을 해줍니다.
만약 중괄호 안의 내용이 짧다면 개행 없이 한 줄로 표현하기도 합니다!
```suggestion
        guard let inputValue = readLine(),
              let menu = Menu(rawValue: inputValue) else { 
              throw GameError.menuNotFound 
              }
```